### PR TITLE
Fixing metadata

### PR DESCRIPTION
**Pull request by msperlin**

### CODE REVIEW

**Comment by msperlin** [comment]
Hi, I justed fixed the python scripts and updated the metadata of the package. I also fixed the CHECK regarding download tests and roxygen version.

**Comment by noamross** [comment]
Excellent, thank you @msperlin! @Myfanwy would you be able to take a look and test that everything runs and makes sense on your end for this?

File: data-raw/01-download-metadata.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Bash script for downloading and parsing metadata files from gutemberg.org
```
Comment (jonthegeek) [inline data-raw/01-download-metadata.sh:1]:
```suggestion
# Bash script for downloading and parsing metadata files from gutenberg.org
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
Just one tiny typo in a comment, as far as I can tell.

I'm going to try to run checks to confirm that the results make sense, but so far it looks plausible, at least!

File: DESCRIPTION [hunk 2]
```diff
@@ -20,7 +20,7 @@ Imports:
     urltools,
     stringr,
     lazyeval
-RoxygenNote: 7.1.1
+RoxygenNote: 7.2.1
```
Comment (jonthegeek) [inline DESCRIPTION:23]:
```suggestion
RoxygenNote: 7.2.1
Encoding: UTF-8
```

Comment (jonthegeek) [reply DESCRIPTION:23]: Specifying encoding to make roxygen2 happier.

**Review by jonthegeek — CHANGES_REQUESTED** [review]
One other thing to make checks happier.

**Review by Myfanwy — APPROVED** [review]
I agree with @jonthegeek's changes. The only other issues I ran in to were that the suggested `attr(gutenberg_metadata, "date_updated")` still returned the "2016-05-05" date of the previous metadata update, even though that attribute does update when I run code in-line from `data-raw/book_metadata.R`.  Also, building with `R CMD build gutenbergr` from the Terminal fails on the intro vignette for me, but I suspect that's user error on my part.  @jonthegeek  or @msperlin see if you can reproduce?

Apologies for any departures from convention(s) - first-time PR reviewer here

**Comment by msperlin** [comment]
> 

Good morning @Myfanwy. I just reproduced the problem with the dates in `attr(gutenberg_metadata, "date_updated")` . To fix it, simply reinstall the package and the dates will show correctly:

image

**Comment by msperlin** [comment]
Your local repository had the previous gutenbergr version installed.

**Comment by msperlin** [comment]
> Also, building with R CMD build gutenbergr from the Terminal fails on the intro vignette for me, but I suspect that's user error on my part. @jonthegeek or @msperlin see if you can reproduce?

Yes, and it worked fine here:

image

I also ranned a local CHECK in Ubuntu 22.04 and it looks good:

image

I'm now implementing a GH CHECK action for Win/MAC and I'll post the results.

**Comment by msperlin** [comment]
> I'm now implementing a GH CHECK action for Win/MAC and I'll post the results.

Unexpectedly, The GH action was not triggered in this PR. But I managed to run a CHECK in a local VM Win machine and it worked fine. I expect a positive result in a MAC, but it would be better if someone can confirm it (I don't have access to a MAC here..).

**Comment by Myfanwy** [comment]
@msperlin after pulling the most recent commits to the fixing-metadata branch, the package builds and installs just fine (I was just missing a dependency before).  R CMD check on the built .tar.gz on my MAC yields 1 note:

```
checking data for non-ASCII characters ... NOTE
  Note: found 13617 marked UTF-8 strings
```
I am confident I can address this this week so I am tempted to merge.  @noamross & @jonthegeek, thoughts?

**Comment by msperlin** [comment]
> @msperlin after pulling the most recent commits to the fixing-metadata branch, the package builds and installs just fine (I was just missing a dependency before). R CMD check on the built .tar.gz on my MAC yields 1 note:

> I am confident I can address this this week so I am tempted to merge. @noamross & @jonthegeek, thoughts?

Great. 

> @msperlin if you give me access permissions on the branch, I can push a change and test the GH action for the MAC?

Sure. I just added you to the repo. Let me know if I can help any further.

**Review by jonthegeek — CHANGES_REQUESTED** [review]
Delete the gutenbergr_0.2.0.tar.gz file (and then we'll squash-and-merge when we merge this to keep it out of the history). Having that file in version control can make it harder for people to get started contributing in the future, and there's no need. We'll probably include it in a *tag* later, but not in the main repo.

**Review by jonthegeek — APPROVED** [review]
This looks good!

Be sure to squash & merge (not just merge) when this one gets merged into master!

**Pull request merged**